src/test/java/com/bluesource/employees/ManagerListInactiveEmployees.java

```diff
@@ -0,0 +1,105 @@
+		 String fullName=null;
+		 int rowNum=0;
+		 
+		 LoginPage loginPage = new LoginPage(getDriver());
+		 Header header = new Header(getDriver());
+		 Employees employees = new Employees(getDriver());
+		 EmployeePage employeePage = new EmployeePage(getDriver());
+		 
+		 // Steps 1&2. Log in to 10.238.243.127 as company.admin
+		 loginPage.AdminLogin();
+		 header.navigateEmployees();
+		 TestReporter.logStep("The page listing employees has been reached.");
+		 // Step 3. Select the first Active employee on the list
+		 rowNum=employees.findFirstActiveInTable();
+		 employees.selectEmployeeFirstname(rowNum, 1);
+		 TestReporter.logStep("Successfully landed on the first active employee's page");
+		 // Step 4. Click the user's Edit button over general info
+		 employeePage.editGeneralInfo();
+		 fullName=employeePage.getFullName();
+		 // Step 5. Click Deactive Employee
+		 employeePage.clickDeactivateEmployee();		 
+		 // Step 6. Click Deactive
+		 employeePage.clickDeactivate();
+		 TestReporter.assertEquals(employees.getSuccessMessage(), successMessage, "Verifying "+fullName+" was successfully made inactive.");
+		 // Step 7. Navigate to the employees page
+		 header.navigateEmployees();
+		 TestReporter.logStep("The page listing employees has been reached.");
+		 // Step 8. Click the add employees button
+		 employees.clickAddEmployee();
+		 TestReporter.logStep("Opened the 'add employees' Modal Popout");
+		 // Step 9. Scroll down to the 'Manager' dropdown list
+		 employees.scrollTo1stManager();
+		 // Step 10. Verify the Employee that was just deactivated is not appearing in the list.
+		 TestReporter.assertFalse(employees.checkManagerOption(fullName), "Asserting "+fullName+" isn't listed as a manager.");
+		 // Step 11. Click the 'Close' button.
+		 employees.clickClose();
+		 // Step 12. Click on the name of the second Employee.
+		 employees.clickSecondName();
+		 TestReporter.logStep("Successfully landed on the second employee's page");
+		 // Step 13. Inside the 'General Info' panel click the 'Edit' button.
+		 employeePage.editGeneralInfo();
+		 // Step 14. Verify in the 'Manager' dropdown list, the Inactive Employee does not appear.
+		 TestReporter.assertFalse(employees.checkManagerOption(fullName), "Asserting "+fullName+" still isn't listed as a manager.");
+		 // Step 15. Click the 'Close' button.
+		 employeePage.clickClose();
+		 // Step 16. Click on 'Employees' in the header.
+		 header.navigateEmployees();
+		 TestReporter.logStep("The page listing employees has been reached.");
+		 // Step 17. Click on the name of the Inactive Employee.
+		 employees.selectEmployeeFirstname(rowNum, 1);
+		 // Step 18. Inside the 'General Info' panel click the 'Edit' button.
+		 employeePage.editGeneralInfo();
+		 // Step 19. Inside the 'Status' dropdown list, select 'Permanent' and click the 'Update Employee' button.
+		 employeePage.selectStatus("Permanent");
+		 employeePage.clickUpdateEmployee();
```

Comment:
You are missing assertions in this test.  You should be asserting that things are successful or not, such as page displaying, success messages displaying, employee displaying in table, employee successfully updated, etc